From: Hideki Yamane <henrich@debian.org>
Date: Sat, 3 Feb 2018 23:02:49 +0900
Subject: Avoid failure on some virtual environment

On some VM, /proc/fb says "xen cirrusdrmfb" and it causes
"[: too many arguments" error.

See https://ci.debian.net/data/autopkgtest/unstable/amd64/s/screenfetch/20180202_195711/log.gz
---
 screenfetch-dev | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/screenfetch-dev b/screenfetch-dev
index 94c412b..c8fd2ff 100755
--- a/screenfetch-dev
+++ b/screenfetch-dev
@@ -1338,7 +1338,7 @@ DetectIntelGPU() {
 		*intel*) gpu=intel ;;
 	esac
 
-        if [ $gpu = intel ]; then
+        if [ "$gpu" = intel ]; then
 		#Detect CPU
 		local CPU=$(uname -p | awk '{print $3}')
 		CPU=${CPU#*'-'}; #Detect CPU number
